lint/verilator_waiver.vlt

```diff
@@ -12,15 +12,24 @@
 `verilator_config
 lint_off -rule PINCONNECTEMPTY
 
+// We have some boolean top-level parameters in e.g. ibex_core_tracing.sv.
+// When building with fusesoc, these get set with defines like
+// -GRV32M=1 (rather than -GRV32M=1'b1), leading to warnings like:
+//
+//   Operator VAR '<varname>' expects 1 bits on the Initial value, but
+//   Initial value's CONST '32'h1' generates 32 bits.
+//
+// This signoff rule ignores errors like this. Note that it only
+// matches when you set a 1-bit value to a literal 1, so it won't hide
+// silly mistakes like setting it to 2.
```

Comment:
I love the fact that my text has propagated through the design, but this is now the third copy :-D Not as part of this PR, but I wonder whether there's a way we could consolidate things to avoid the repetition?